codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameReader.java

```diff
@@ -496,6 +492,12 @@ public void processFragment(boolean endOfHeaders, ByteBuf fragment,
         resetHeadersContinuationIfEnd(flags.endOfHeaders());
     }
 
+    /**
+     * If received {@code true} from {@link Http2Flags#endOfHeaders()} ,
+     * close {@link #headersContinuation} to represent current state is not processing header block,
+     * And can start receiving frame other than CONTINUATION frame
+     * @param endOfHeaders
```

Comment:
I'm not sure these comments are necessary. The method name seems pretty straightforward, the comments could use some rework, and it the method is private. Can we just remove these comments?